.github/workflows/release.yml

```diff
@@ -279,11 +279,23 @@ jobs:
           # Add additional line before the content
           RELEASE_NOTES="✅ New *${{ github.event.repository.name }}* Release [${{ needs.version-tag.outputs.version }}](${RELEASE_URL}) $(date)\n\n${RELEASE_NOTES}\n\nCommit Details:\n${new_fix_notes}"
 
-          # Combine existing content with new fix notes
```

Comment:
- The removal of extra HTML using `sed` commands may introduce unintended side effects if the content contains legitimate HTML tags. Ensure that this approach does not strip necessary formatting unintentionally.